### ISSUE TITLE
No sidebar on WDS project page

### ISSUE THREAD

**Issue by awfrancisco** [18F/18f.gsa.gov#2491]
## Expected Behavior
The WDS project page would have a sidebar with links to github and the main site. 
https://18f.gsa.gov/what-we-deliver/us-web-design-standards/

## Actual Behavior
No sidebar, even though the information seems to be in the front matter.

## This is issue is done when:
- [ ] Accurate WDS info appears on the right sidebar of the project page

@elainekamlley

**Comment by gemfarmer** [comment]
@awfrancisco this looks like is caused by incorrect variable names [here](https://github.com/18F/18f.gsa.gov/blob/master/_layouts/project-page.html#L45-L46)

That if statement is currently:

```
{% if page.agency or
          page.product_url or
          page.github_url or
          any_clients or
          page.start_date or
          page.expiration_date or
          page.resources
    %}
```

but should be:

```
{% if page.agency or
          page.project_url or
          page.github_repo or
          any_clients or
          page.start_date or
          page.expiration_date or
          page.resources
    %}
```

**Comment by elainekamlley** [comment]
fixed with #2492
